GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -171,6 +176,59 @@ private void ReloadCommitInfo()
 
             if (AppSettings.CommitInfoShowContainedInTags)
                 ThreadPool.QueueUserWorkItem(_ => loadTagInfo(_revision.Guid));
+
+
+            commitSignPicture.Visible = tagSignPicture.Visible = AppSettings.ShowGpgInformation;
+
+            if (AppSettings.ShowGpgInformation)
+            {
+                /* Initialization in constructior will throw exception because of non initialized Module. */
+                if (_gpgController == null)
+                {
+                    _gpgController = new GitGpgController(Module);
```

Comment:
@RussKie 
`Module` is null in when running construction, so I've put the creation here.
No async/await pattern here, and I see it slows the UI, this because of entire `ReloadCommitInfo` shall be async.